definitions/firefox_ios.toml

```diff
@@ -225,3 +232,9 @@ from_expression = """(
 experiments_column_type = "simple"
 client_id_column = "client_id"
 submission_date_column = "submission_date"
+
+[data_sources.new_profile_activation]
+from_expression = "`moz-fx-data-shared-prod.firefox_ios.new_profile_activation`"
+experiments_column_type = "simple"
+client_id_column = "client_id"
```

Comment:
`client_id_column` and `submission_date_column` are both [default values](https://github.com/mozilla/metric-config-parser/blob/main/metric_config_parser/data_source.py#L54-L55) so I think they could be left off